components/Layout.vue

```diff
@@ -58,82 +56,90 @@
           class="ml-auto text-right"
         >
           <dt
-            class="font-display text-sm font-medium text-slate-900 dark:text-white"
+            class="font-display text-sm font-medium text-slate-900"
           >
             Next
           </dt>
           <dd class="mt-1">
             <NuxtLink
               :href="nextPage._path"
-              class="text-base font-semibold text-slate-500 hover:text-slate-600 dark:text-slate-400 dark:hover:text-slate-300"
+              class="text-base font-semibold text-slate-500 hover:text-slate-600"
             >
               {{ nextPage.title }} <span aria-hidden="true">&rarr;</span>
             </NuxtLink>
           </dd>
         </div>
       </dl>
     </div>
-    <div class="hidden xl:sticky xl:top-[4.5rem] xl:-mr-6 xl:block xl:h-[calc(100vh-4.5rem)] xl:flex-none xl:overflow-y-auto xl:py-16 xl:pr-6">
-      <nav
-        aria-labelledby="on-this-page-title"
-        class="w-56"
+    <nav
+      class="sticky top-[4.5rem] flex flex-col w-full px-4 bg-white lg:max-h-page lg:self-start lg:bg-transparent lg:backdrop-blur-none lg:flex-none lg:overflow-y-auto lg:py-16 lg:col-span-1"
```

Comment:
Minor - a shadow here might be nice to give the TOC dropdown some depth. It's hard to tell where the dropdown ends and the content starts.

<img width="982" alt="image" src="https://user-images.githubusercontent.com/50732795/194989649-e6d0b339-e3b9-47c5-a358-808c28a04051.png">

Reply:
I opted against this as it doesn't look great next to the drop shadow being applied to the header
<img width="501" alt="image" src="https://user-images.githubusercontent.com/46909693/195004735-c3f2bec1-d61a-4132-b9f9-cc0b82a4e5ae.png">

Reply:
A small drop shadow looks better - perhaps this is ok enough to leave in
<img width="507" alt="image" src="https://user-images.githubusercontent.com/46909693/195005416-c256b3c0-ce6a-4528-9e16-fe3e60a2526f.png">

Reply:
Yep, I like the small drop shadow 😄